src/clientiface.cpp

```diff
@@ -242,7 +246,6 @@ void RemoteClient::GetNextBlocks (
 				FOV setting. The default of 72 degrees is fine.
 			*/
 
-			float camera_fov = (72.0*M_PI/180) * 4./3.;
```

Comment:
Just checking something, here the default FOV 72deg is multiplied by 4/3, i assume to get horizontal FOV in radians.
Now that you're getting FOV directly is this * 4/3 still being done?
I'm assuming that the player accessible FOV setting is vertical FOV.